README.md

```diff
@@ -318,6 +318,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [LeakCheck.io](https://leakcheck.io/) - Make sure your credentials haven't been compromised
 - [crackstation.net](https://crackstation.net/) -Massive pre-computed lookup tables to crack password hashes
 - [HashKiller](https://hashkiller.io/listmanager) - Pre-cracked Hashes, easily searchable
+- [Hudson Rock](https://www.hudsonrock.com/threat-intelligence-cybercrime-tools) - Check if a username, email, or domain was compromised in global Infostealer attacks
```

Comment:
Maybe this description would be better: `Use Hudson Rock’s free cybercrime intelligence tools to learn how compromised credentials are impacting your business`

Reply:
ok, waiting for the changes